MapboxNavigation/UIView.swift

```diff
@@ -39,7 +39,7 @@ class RippleLayer: CAReplicatorLayer {
         }
     }
     var rippleRadius: CGFloat = 100
-    var rippleColor: UIColor = UIColor.white
+    var rippleColor: UIColor = .red
```

Comment:
Should we make this color stylable? The developer has the option to make the background dark, right? Red against a dark background doesn’t work very well.

Reply:
Yeah let's do this afterwards.